smithy-aws-cloudformation-traits/src/test/resources/software/amazon/smithy/aws/cloudformation/traits/test-service.smithy

```diff
@@ -51,6 +52,7 @@ operation GetFooOperation {
 
 structure GetFooRequest {
     @required
+    @cfnDefaultValue
```

Comment:
Is this trait useful without the default value being specified here? Is the actual default value supposed to be deferred to a runtime determination by clients?

Reply:
Yes, we just need to be able to mark a member as the default value. This is crucial for drift detection especially if the configuration of a monitored resource has changed from a pre-defined configuration

Reply:
The specific default value should not be in the Smithy model as well?

Reply:
As discussed offline: We don't need the actual default value to be passed into the trait. The main behavior we need with this trait is the ability to annotate a member as default so we know the specific members/properties that must match for drift detection.